cyctl/internal/update/modules.go

```diff
@@ -0,0 +1,91 @@
+package update
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/cyclops-ui/cyclops/cyclops-ctrl/api/v1alpha1/client"
+	"github.com/cyclops-ui/cycops-cyctl/internal/kubeconfig"
+	"github.com/spf13/cobra"
+	apiextensionv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+var (
+	updateModuleExample = `# updates the module,takes [module-name] as an argument with flags --key and --value
+# to update replicas for a module named test 
+cyctl update module test --key="scaling.replicas" --value=3
+	`
+)
+
+// updates the given module from cyclops API
+// currently supports updating replica count for a module
+func updateModule(clientset *client.CyclopsV1Alpha1Client, moduleName, key string, value int) {
```

Comment:
Can you make the value type `interface`?

Reply:
done